tests/unittest/benchmark_runner/common/workload_flavors/golden_files/func_ci_vdbench_kata_OCS_PVC_False/vdbench_pod.yaml

```diff
@@ -0,0 +1,102 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: benchmark-operator
```

Comment:
Namespace should be benchmark-runner in all the below vdbench places

Reply:
Fixed.